tests/conftest.py

```diff
@@ -52,3 +55,23 @@ def factory(**config):
         InvenioSearch(app)
         return app
     return factory
+
+
+@pytest.fixture(scope="function")
+def superuser_role_need(db):
+    """Store 1 role with 'superuser-access' ActionNeed.
+
+    WHY: This is needed because expansion of ActionNeed is
+         done on the basis of a User/Role being associated with that Need.
+         If no User/Role is associated with that Need (in the DB), the
+         permission is not expanded.
```

Comment:
Comment: It is expanded, but to *nothing* so its empty in the end.

Reply:
:heavy_check_mark: